ci-operator/config/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main.yaml

```diff
@@ -0,0 +1,85 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+binary_build_commands: NO_DOCKER=1 make build
+build_root:
+  from_repository: true
+canonical_go_repository: sigs.k8s.io/cluster-api-provider-ibmcloud
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8
+  to: ibmcloud-machine-controllers
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: unit
+  commands: NO_DOCKER=1 make test
+  container:
+    from: src
+- as: goimports
+  commands: IS_CONTAINER=TRUE make goimports
+  container:
+    from: src
+- as: govet
+  commands: IS_CONTAINER=TRUE make vet
+  container:
+    from: src
+- as: generate
+  commands: IS_CONTAINER=TRUE NO_DOCKER=1 make generate
+  container:
+    from: src
+- as: golint
+  commands: IS_CONTAINER=TRUE make lint
+  container:
+    from: src
+- as: e2e-ibmcloud
+  steps:
+    cluster_profile: ibmcloud
+    workflow: openshift-e2e-ibmcloud
+- as: e2e-ibmcloud-operator
+  steps:
+    cluster_profile: ibmcloud
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
+      from: stable:cluster-api-actuator-pkg
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-ibmcloud
+- as: e2e-ibmcloud-upgrade
+  steps:
+    cluster_profile: ibmcloud
+    workflow: openshift-upgrade-ibmcloud
+- as: e2e-ibmcloud-serial
+  steps:
+    cluster_profile: ibmcloud
+    workflow: openshift-e2e-ibmcloud-serial
```

Comment:
i'm kinda wondering if we need to replace this with the e2e-aws stuff from the release-4.9.yaml, at least until we have the ibmcloud ready for testing.

Reply:
done
@elmiko any suggested edits for `e2e-ibmcloud` references within the following presubmit yamls?
Should I convert them over to `e2e-aws` or ?
```
/ci-operator/jobs/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main-presubmits.yaml
/ci-operator/jobs/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-release-4.9-presubmits.yaml
```

Reply:
i think so (sorry if i missed those), imo i think we should just make this PR do only the e2e-aws tests across the board. lets just make it simple, then when the ibmcloud is ready as a platform we can add on the other tests. this way we can at least get the basic _smoke test_ working and then proceed.